src/GeoJsonPages/GeoJsonContentHandler.php

```diff
@@ -35,18 +39,20 @@ protected function fillParserOutput(
 	) {
 		'@phan-var GeoJsonContent $content';
 
-		if ( !$cpoParams->getGenerateHtml() || !$content->isValid() ) {
-			$parserOutput->setText( '' );
-			return;
-		}
+		parent::fillParserOutput( $content, $cpoParams, $parserOutput );
```

Comment:
Why did you add this call?

Reply:
this is to set the correct parserOutput text by calling the same method in the parent class (JsonContentHandler)